templates/terraform/examples/ha_vpn_gateway_basic.tf.erb

```diff
@@ -0,0 +1,10 @@
+resource "google_compute_ha_vpn_gateway" "ha_gateway" {
+  provider = "google-beta"
+  name     = "<%= ctx[:vars]['ha_vpn_gateway_name'] %>"
+  network  = "${google_compute_network.network1.self_link}"
+}
+
+resource "google_compute_network" "network1" {
```

Comment:
also add `auto_create_subnetworks = false` to make it spin up faster (so it doesn't have to spin up the network + a bunch of subnetworks